integration/integrate-with-splunk.adoc

```diff
@@ -7,19 +7,14 @@ include::modules/common-attributes.adoc[]
 toc::[]
 
 [role="_abstract"]
-If you are using link:https://www.splunk.com/[Splunk], you can forward alerts from {product-title} to Splunk and view vulnerability and compliance related data from within Splunk.
+If you are using link:https://www.splunk.com/[Splunk], you can forward alerts from {product-title} to Splunk and view violations, vulnerability detection, and compliance related data from within Splunk.
 
 Depending on your use case, you can integrate {product-title} with Splunk by using the following ways:
 
 * By xref:../integration/integrate-with-splunk.adoc#integrate-splunk-http-event-collector[using an HTTP event collector] in Splunk
 ** Use the event collector option to forward alerts and audit log data
-* By xref:../integration/integrate-with-splunk.adoc#integrate-splunk-add-on[using the StackRox Kubernetes Security Platform add-on]
-** Use the add-on to pull vulnerability detection and compliance data into Splunk
-+
-[NOTE]
-====
-The StackRox Kubernetes Security Platform add-on is only available if you are using {product-title} version 3.0.51.0 or newer.
-====
+* By xref:../integration/integrate-with-splunk.adoc#integrate-splunk-add-on[using the {product-title} add-on]
+** Use the add-on to pull violations, vulnerability detection, and compliance data into Splunk.
```

Comment:
This one adds the period (which is correct since it's a full sentence), but the above sentence on line 15 doesn't. Can you add it there too, for consistency?